neon_local/src/main.rs

```diff
@@ -159,6 +159,20 @@ fn main() -> Result<()> {
                 .about("Create a new blank timeline")
                 .arg(tenant_id_arg.clone())
                 .arg(branch_name_arg.clone()))
+            .subcommand(App::new("import")
+                .about("Import timeline from basebackup directory")
+                .arg(tenant_id_arg.clone())
+                .arg(timeline_id_arg.clone())
+                .arg(Arg::new("node-name").long("node-name").takes_value(true)
+                    .help("Name to assign to the imported timeline"))
+                .arg(Arg::new("base-tarfile").long("base-tarfile").takes_value(true)
+                    .help("Basebackup tarfile to import"))
+                .arg(Arg::new("base-lsn").long("base-lsn").takes_value(true)
+                    .help("Lsn the basebackup starts at"))
+                .arg(Arg::new("wal-tarfile").long("wal-tarfile").takes_value(true)
+                    .help("Wal to add after base"))
+                .arg(Arg::new("end-lsn").long("end-lsn").takes_value(true)
+                    .help("Lsn the basebackup ends at")))
```

Comment:
For the sake of tests I'm using `neon_local` as the client interface, but we will need a binary that talks directly to deployed pageservers. I'll make one after this PR, unless there's some `psql` trick for streaming a file using copy protocol.

Reply:
So this API allows us to upload existing basebackup.tar

Maybe just add another endpoint to `page_service.rs`:
 `"import fullbackup pageserver_connstr tenant_id timeline_id lsn"` 
 that will connect another pageserver directly, issue `"fullbackup tenant_id timeline_id lsn"` request and receive the data via copy protocol?

Reply:
let's leave this for another PR. It's an extra feature